### DISCUSSION TITLE
How do I install this mod?

### DISCUSSION THREAD

**Discussion by jh0325** [TitanNano/TS4ControlAnySim#42] [Q&A] [answered]
im sorry im kinda new...please tell me how to use this. im completely lost. ive tried going into actions on a sim but i dont see anything about controlling them

**Comment by TitanNano** [comment, marked as answer]
The mod should be extracted and placed inside your mods folder. This should look somewhat like this:

- `.../The Sims 4/Mods/ControlAnySim/ControlAnySim.package`
- `.../The Sims 4/Mods/ControlAnySim/ControlAnySim.ts4script`

You then have make sure that you enabled mods in your game settings and also allowed script mods.

**Comment by aemegi** [comment]
I think what OP was trying to ask, which I am wondering as well, is how do you set sims to be an NPC?
I downloaded this mod then shift+clicked a sim with testing cheats on, and saw the remove from household button, but later realized that is a game cheat not your mod.  And this feature does not quite do what I would like it to,
Thank you for making this by the way, it is exactly what I was looking for, I made a boarding house and didn't really want to play with the owner and all the other tenants, I just want control over one of them. :)

**Reply by TitanNano** [reply]
When you click on a sim there should be an interaction group called "Actions". In this group you should have interactions called "Add To Group" and the like. These are from Maxis, but in addition to them, you should now have an interaction called "Control Sim".

**Reply by TitanNano** [reply]
> how do you set sims to be an NPC?

To answer this, in the "Actions" group should also be an interaction called "Turn Into Household NPC". This action is only available for actual household members though. You will have to add sims to the household before you can use this interaction.

**Reply by aemegi** [reply]
Thanks! I will have to try it.
